pkg/clusterapi/system.go

```diff
@@ -492,7 +510,7 @@ func (c *system) runController(ctx context.Context, ct *controller) error {
 
 	// Run the controller and store its state.
 	logrus.Infof("Running process: %s with args %v", ct.Name, ct.Args)
-	if err := pr.Start(ctx, os.Stdout, os.Stderr); err != nil {
+	if err := pr.Start(ctx, io.MultiWriter(os.Stdout, c.logFile), io.MultiWriter(os.Stderr, c.logFile)); err != nil {
```

Comment:
```suggestion
	loggerFileWriter := io.MultiWriter(logrus.StandardLogger().WriterLevel(logrus.DebugLevel), c.logFile)
	if err := pr.Start(ctx, loggerFileWriter, loggerFileWriter); err != nil {
```
Something like this may do what you were talking about: hide controller logs with log-level info and show them with log-level debug. Wdyt? 

This is a simple example. Not sure if you want to use a different log level for the stderr output?

Reply:
I like this. In my tests, all the logging is coming from `stderr` so I'll use the same writer for both.